features/admin.approval-workflows.v1/components/create/general-approval-workflow-details-form.tsx

```diff
@@ -95,7 +95,7 @@ const GeneralApprovalWorkflowDetailsForm: ForwardRefExoticComponent<RefAttribute
             const currentValuesRef: any = useRef(null);
 
             const dispatch: Dispatch = useDispatch();
-            const { t } = useTranslation();
+            const { t } = useTranslation([ "approvalWorkflows" ]);
```

Comment:
Is it necessary to add this namespace here explicitly? This should already be available for the console. Please check other places as well.

Reply:
Let's add the newly added i18n namespace to the console.
You can follow: https://github.com/wso2/identity-apps/commit/fbc6986ba5e65c5ed008b7fe5358d1a1eef7d17f (This is a similar thing done for My Account)